deploy/helm/templates/cluster-role.yaml

```diff
@@ -15,6 +15,7 @@ rules:
       - nodes
       - nodes/spec
       - nodes/proxy
+      - nodes/metrics
```

Comment:
this is needed to scrape cAdvisor/kubelet metrics directly, otherwise it throws "Forbidden 403" errors